src/main/groovy/org/mockito/release/internal/gradle/RecentContributorsFetcherTask.java

```diff
@@ -53,73 +54,105 @@ public void setReleaseNotesData(File releaseNotesData) {
-        ContributorsSet contributors = contributorsProvider.mapContributorsToGitHubUser(authorNames, fromRev, toRevision);
-
-        ContributorsSerializer contributorsSerializer = Contributors.getLastContributorsSerializer(contributorsFile);
-        contributorsSerializer.serialize(contributors);
-    }
-
-    private static Collection<String> getAuthorNames(Collection<ReleaseNotesData> data) {
-        Set<String> authors = new HashSet<String>();
-        for (ReleaseNotesData d : data) {
-            for (Contribution c : d.getContributions().getContributions()) {
-                authors.add(c.getAuthorName());
-            }
-        }
-        return authors;
+    /**
+     * See {@link ReleaseConfiguration.GitHub#getRepository()}
+     */
+    public String getRepository() {
+        return repository;
     }
 
+    /**
+     * See {@link #getRepository()}
+     */
     public void setRepository(String repository) {
         this.repository = repository;
     }
 
+    /**
+     * See {@link ReleaseConfiguration.GitHub#getReadOnlyAuthToken()}
+     */
+    public String getReadOnlyAuthToken() {
+        return readOnlyAuthToken;
+    }
+
+    /**
+     * See {@link #getReadOnlyAuthToken()}
+     */
     public void setReadOnlyAuthToken(String readOnlyAuthToken) {
         this.readOnlyAuthToken = readOnlyAuthToken;
     }
 
-    public void setToRevision(String toRevision) {
-        this.toRevision = toRevision;
+    /**
+     * Revision from which to iterate commits to find contributors
+     * See also {@link #getToRevision()}
+     */
+    public String getFromRevision() {
+        return fromRevision;
     }
 
+    /**
+     * See {@link #getFromRevision()}
+     */
     public void setFromRevision(String fromRevision) {
         this.fromRevision = fromRevision;
     }
 
-    public void setContributorsFile(File contributorsFile) {
-        this.contributorsFile = contributorsFile;
+    /**
+     * Revision to which to iterate commits to find contributors.
+     * See also {@link #getFromRevision()}
+     */
+    public String getToRevision() {
+        return toRevision;
     }
 
-    public String getRepository() {
-        return repository;
+    /**
+     * See {@link #getToRevision()}
+     */
+    public void setToRevision(String toRevision) {
+        this.toRevision = toRevision;
     }
 
-    public String getReadOnlyAuthToken() {
-        return readOnlyAuthToken;
+    /**
+     * Output file to serialize contributors data
+     */
+    public File getOutputFile() {
+        return outputFile;
     }
 
-    public String getFromRevision() {
-        return fromRevision;
+    /**
+     * See {@link #getOutputFile()}
+     */
+    public void setOutputFile(File outputFile) {
+        this.outputFile = outputFile;
     }
 
-    public String getToRevision() {
-        return toRevision;
+    @TaskAction
+    public void fetch() {
```

Comment:
I moved the setters up for consistency with other code. I think you are right that it is more important than the setters so it should probably by up :) For now I'll leave things consistent. Your feedback was noted!